### PR TITLE
Add quotes to TTA promo page

### PR DESCRIPTION
**Pull request by ethax-ross**
### Trello card

[Trello-4012](https://trello.com/c/rH11nJte/4012-consider-creating-new-blog-promo-block-for-teacher-training-advisers-tta-page)

### Context

We want to add a list of quotes to the TTA promo page to replace the accordion component that was recently removed.

### Changes proposed in this pull request

- Remove unused TTA page partials

- Add quotes to the TTA promo page

- Add QuoteListComponent

We want to display a list of quotes on some of our content pages; this is similar to the `PhotoQuoteListComponent` but without the photos and with the heading half overlapping the containing box. I initially tried to combine the two quote list components but it became awkward due to the overlapping heading on the non-photo variant, so I've kept them separate.

### Guidance to review

[TTA page](https://review-get-into-teaching-app-3068.london.cloudapps.digital/teacher-training-advisers)

| Desktop      | Mobile |
| ----------- | ----------- |
| Screenshot 2023-01-09 at 10 58 15      | Screenshot 2023-01-09 at 10 58 35       |

### CODE REVIEW

**Review by gazcarr — DISMISSED** [review]

**Comment by EllieNodder** [comment]
@gemmadallmandfe do we need to run these past Andy first?

File: app/components/content/quote_list_component.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  private
+
+    def before_render
+      validate!
+    end
+
+    def validate!
+      quotes.each_with_index do |quote, idx|
+        %i[heading text accreditation].each do |required_key|
```
Comment (asatwal) [inline app/components/content/quote_list_component.rb:19]: Nitpick: I would have the array of symbols as a constant:
``` Ruby
REQUIRED_KEYS = %i[heading text accreditation].freeze
```
With the constant being defined at the top of the module. Makes it a bit more readable. But that's just my preference and please feel free to ignore me -:)

**Review by asatwal — APPROVED** [review]
👍  LGTM

File: app/components/content/quote_list_component.rb (hunk 1 above)
Comment (ethax-ross) [reply app/components/content/quote_list_component.rb:19]: Yeah agreed, that is cleaner - pushed an update

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Review by philspray — APPROVED** [review]
I agree, looks good.

**Pull request merged**